drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -638,25 +638,11 @@ def _expand_static_url(self, url):
     def _get_user_state(self):
         """ Get all user-specific data, and any applicable feedback """
         item_state = self._get_item_state()
-        for item_id, item in item_state.iteritems():
-            # If information about zone is missing
-            # (because problem was completed before a11y enhancements were implemented),
-            # deduce zone in which item is placed from definition:
-            if item.get('zone') is None:
-                valid_zones = self._get_item_zones(int(item_id))
-                if valid_zones:
-                    # If we get to this point, then the item was placed prior to support for
-                    # multiple correct zones being added. As a result, it can only be correct
-                    # on a single zone, and so we can trust that the item was placed on the
-                    # zone with index 0.
-                    item['zone'] = valid_zones[0]
-                else:
-                    item['zone'] = 'unknown'
-
-            # In assessment mode, if item is placed correctly and than the page is refreshed, "correct"
-            # will spill to the frontend, making item "disabled", thus allowing students to obtain answer by trial
-            # and error + refreshing the page. In order to avoid that, we remove "correct" from an item here
-            if self.mode == self.ASSESSMENT_MODE:
+        # In assessment mode, if item is placed correctly and than the page is refreshed, "correct"
+        # will spill to the frontend, making item "disabled", thus allowing students to obtain answer by trial
+        # and error + refreshing the page. In order to avoid that, we remove "correct" from an item here
```

Comment:
@mtyaka @mtyaka Typo: "than" should be "then"

And I know you just copied an existing comment, but I'm not sure what

> making item "disabled", thus allowing students to obtain answer by trial and error ...

means (esp. the first part)? It would be great if you could clarify :)